spec/controllers/v0/sessions_controller_spec.rb

```diff
@@ -120,6 +120,14 @@
           expect(post(:saml_callback)).to redirect_to(Settings.saml.relay + '?auth=fail')
           expect(response).to have_http_status(:found)
         end
+        it 'increments the failed statsd counter' do
+          once = { times: 1, value: 1 }
+          expect do
+            post(:saml_callback)
+          end.to trigger_statsd_increment(
+            described_class::STATSD_LOGIN_FAILED_KEY, tags: ['error:auth_too_early'], **once
+          ) & trigger_statsd_increment(described_class::STATSD_LOGIN_TOTAL_KEY, **once)
+        end
```

Comment:
Ha!  @jkassemi and I were trying to get `.and` to work and failed.  I'll give it another go.